### PR TITLE
Update frontend for matches + fetching api

### PR DESCRIPTION
**Pull request by alexsc23**

### CODE REVIEW

**Comment by alexsc23** [comment]
Close #16

File: web/frontend/src/App.svelte [hunk 1]
```diff
@@ -6,19 +6,40 @@
...
-    return 0;
+    fetch("http://dayo-project.herokuapp.com/api/v1/participants")
+    .then(response => response.json())
+    .then(data => {
+      participants = data["data"];
+      participants = participants.filter(participant => {
+        return participant.season === current_season;
+      })
+    }).catch(error => {
+      console.log(error);
```
Comment (xTomsko) [inline web/frontend/src/App.svelte]: Don't return error. Use custom error instead. Error will leak database details

File: web/frontend/src/App.svelte [hunk 2]
```diff
@@ -6,19 +6,40 @@
...
+  onMount(async () => {
+    fetch("http://dayo-project.herokuapp.com/api/v1/matches")
+    .then(response => response.json())
+    .then(data => {
+      matches = data["data"];
+      matches = matches.filter(matches_obj => {
+        return matches_obj.season === current_season && matches_obj.episode === newest_episode;
+      })
+      matches = matches[0]["matches"]
+    }).catch(error => {
```
Comment (xTomsko) [inline web/frontend/src/App.svelte]: Here as well

**Review by xTomsko — CHANGES_REQUESTED** [review]

File: web/frontend/src/App.svelte (hunk 1 above)
Comment (xTomsko) [reply web/frontend/src/App.svelte]: We might have to check all files. I am sure we committed a few of those before but we should change them imo

**Review by Lesoge — APPROVED** [review]

**Review by xTomsko — APPROVED** [review]

**Pull request merged**